portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,47 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ani's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+
+
+     <div class="navigation">      
```

Comment:
I think `id` is more appropriate than `class` here and in the other html files since the navigation is unique for each web page.

Reply:
Also I think there's an extra leading space on this line.